components/apimgt/org.wso2.carbon.apimgt.rest.api/src/gen/java/org/wso2/carbon/apimgt/rest/api/dto/APIDTO.java

```diff
@@ -306,7 +307,7 @@ public String toString()  {
     sb.append("  context: ").append(context).append("\n");
     sb.append("  version: ").append(version).append("\n");
     sb.append("  provider: ").append(provider).append("\n");
-    sb.append("  swagger: ").append(swagger).append("\n");
+    sb.append("  apiDefinition: ").append(apiDefinition).append("\n");
     sb.append("  status: ").append(status).append("\n");
     sb.append("  responseCaching: ").append(responseCaching).append("\n");
     sb.append("  cacheTimeout: ").append(cacheTimeout).append("\n");
```

Comment:
Please note that all the files in components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/gen including DTO classes get auto-generated using swagger-to-cxf plugin based on the modifications done for api.yaml